src/main/java/com/alkemy/ong/service/impl/CommentServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package com.alkemy.ong.service.impl;
+
+import com.alkemy.ong.dto.CommentDto;
+import com.alkemy.ong.mapper.CommentMapper;
+import com.alkemy.ong.model.Comment;
+import com.alkemy.ong.repository.CommentRepository;
+import com.alkemy.ong.repository.NewsRepository;
+import com.alkemy.ong.repository.UserRepository;
+import com.alkemy.ong.service.CommentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+
+@Service
+public class CommentServiceImpl implements CommentService {
+
+	@Autowired
+	private CommentMapper mapper;
+	@Autowired
+	private CommentRepository repository;
+	@Autowired
+	private NewsRepository newsRepository;
+	@Autowired
+	private UserRepository userRepository;
+
+	@Transactional
+	@Override
+	public Optional<CommentDto> saveComment(CommentDto dto) {
+		if ( newsRepository.existsById(dto.getNewsId()) &&
```

Comment:
Since is not in the acceptance criteria could we do not validate the existence please?